src/20_Components/amp-fx-flying-carpet.html

```diff
@@ -0,0 +1,77 @@
+
+  For best experience of this example, use a mobile device or choose a mobile device layout using the Chrome console.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!-- Import the `amp-fx-flying-carpet` component in the header. -->
+  <script async custom-element="amp-fx-flying-carpet" src="https://cdn.ampproject.org/v0/amp-fx-flying-carpet-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!--
+     #### Basic Usage
+  -->
+  <!--
+
+    `amp-fx-flying-carpet` elements must be positioned after the first viewport
+     so make sure to place enough content before the element to cover the targeted viewport sizes.
+
+     The text that follows is to ensure that the `amp-fx-flying-carpet` is placed after the first viewport.
+  -->
+    <p>
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
+    </p>
+
+  <!--
+    `amp-fx-flying-carpet` can be used to display images.
+    Use `height` parameter to specify the height of the flying carpets "window".
+
+    Use vw/vh to size elements inside the `amp-fx-flying-carpet` relative to the size of the viewport. This will allow the best scrolling effect having the element using the entire viewport. 
+  -->
+  <amp-fx-flying-carpet height="200px">
```

Comment:
I can add another amp-fx-flying carpet displaying an ad (use it as first example and maybe keep the one with image as a second one in the sample file). Do you have an ad which takes the entire viewport? The one you linked is a bit small and you can see a lot of grey background. I think we could demonstrate a better effect if we have an ad bigger and while scrolling we discover more of the add. This was one of the reason we picked an image.